lab04/sections/discussion.tex

```diff
@@ -1,2 +1,10 @@
 \section{Discussion}\label{sec:discussion}
-Analysis and interpretation of data.
+We confirmed frequency analysis through the Laplace transform is a valid and efficient way to analyze the response of a circuit to inputs of particular or varying frequencies. 
```

Comment:
There's not really any frequency analysis occurring. You could think of negative gain as a 180deg phase shift but that's not how we present the analysis.

The laplace domain circuit is analyzed with impedances and algebraic operations rather than integro-differential operations.